src/reducers/books.js

```diff
@@ -0,0 +1,23 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+
+const initialState = {
+  books: [
+    { bookId: Math.ceil(Math.random()), title: 'Understanding in C++', category: 'Programming' },
```

Comment:
* [ ] Math.random() does not guarantee a unique number generation, random numbers are necessarily unique hence the use of packages such as `short-id`, `UUID` etc.. are ideal for robust and efficient identification.